_posts/2024-10-21-Boosting-OpenSearch-Development-on-MacOS-with-Apple-Silicon.md

```diff
@@ -0,0 +1,234 @@
+---
+layout: post
+title:  Streamline your OpenSearch development workflow on macOS with Apple silicon
+authors:
+- zhujiaxiang
+- sisurab
+- kolchfa
+date: 2024-10-21
+categories:
+  - releases
+  - technical-posts
+excerpt: We're excited to announce that the OpenSearch core snapshot tarball artifact, built specifically for macOS ARM64 machines, is now publicly available. If you're developing on the latest Mac computers with Apple silicon, we encourage you to give it a try and experience the streamlined workflow.
+meta_keywords: opensearch open source, MacOS, ARM64, Apple Silicon, developer friendly, architecture support, platform support
+meta_description: Learn how to simplify your OpenSearch development workflow on macOS with Apple silicon
```

Comment:
@peterzhuamazon  - Updated meta: 
_**meta_keywords:**_  OpenSearch core snapshot,  macOS ARM64, OpenSearch Dashboards, development workflow on macOS with Apple silicon

_**meta_description:**_  Learn how to access the new OpenSearch core snapshot artifact for macOS ARM64 and use it to simplify your development process and streamline your workflow on Mac computers with Apple silicon.

Reply:
Updated now.